ANMN/LTSP/TSaggregator/TSagg_metadata.json

```diff
@@ -163,26 +161,26 @@
         "CHLF": {
             "ancillary_variables": "CHLF_quality_control",
             "comment": "Artificial chlorophyll data",
-            "_FillValue": 999999.0, 
+            "_FillValue": 999999.0,
             "long_name": "mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water",
             "units": "mg m-3",
             "valid_max": 100.0,
             "valid_min": 0.0
             },
         "CHLF_quality_control": {
             "_Unsigned": "false",
-            "_FillValue": 99, 
+            "_FillValue": 99,
             "flag_meanings": "No_QC_performed Good_data Probably_good_data Bad_data_that_are_potentially_correctable Bad_data Value_changed Not_used Not_used Not_used Missing_value",
-            "flag_values": "0, 1, 2, 3, 4, 5, 6, 7, 8, 9", 
+            "flag_values": "0, 1, 2, 3, 4, 5, 6, 7, 8, 9",
             "long_name": "quality flag for mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water",
             "quality_control_conventions": "IMOS standard flags",
-            "quality_control_global": "", 
+            "quality_control_global": "",
             "quality_control_global_conventions": "Argo reference table 2a (see http://www.cmar.csiro.au/argo/dmqc/user_doc/QC_flags.html), applied on data in position only (between global attributes time_deployment_start and time_deployment_end)"
             },
         "CHLU": {
             "comment": "Artificial chlorophyll data",
-            "applied_offset": -10.135297, 
-            "long_name": "mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water", 
+            "applied_offset": -10.135297,
```

Comment:
applied_offset??? Are you sure that was from CHLU? (and not PRES_REL)?

Reply:
fixed